src/main/scala/com/cognite/sdk/scala/common/update.scala

```diff
@@ -49,6 +49,7 @@ trait UpdateById[R <: WithId[Long], U, F[_]] extends WithRequestSession[F] with
 
 object UpdateById {
   implicit val assetUpdateEncoder: Encoder[AssetUpdate] = deriveEncoder
+  implicit val labelsOnUpdateEncoder: Encoder[LabelsOnUpdate] = deriveEncoder
```

Comment:
@hakontro Can you please have a look at this change when you have time? I could not understand why we have an implicit encoder definition only for AssetUpdate in update objects? And I needed to add a new dataType for label update on assets.

Reply:
Seems to be leftovers from old code! I can compile without those two Encoders, so try that and see if it still works.